src/leaflike/bookmarks/routes.clj

```diff
@@ -1,32 +1,34 @@
 (ns leaflike.bookmarks.routes
   (:require [leaflike.bookmarks.core :as bm-core]
             [leaflike.middlewares :refer [with-auth-middlewares]]
+            [leaflike.layout :refer [application]]
             [ring.util.response :as res]
-            [buddy.auth :refer [authenticated?]]))
+            [buddy.auth :refer [authenticated?]]
+            [leaflike.bookmarks.views :as views]))
 
 (defn create
   [request]
   (res/response (bm-core/create request)))
 
-(defn list-all
+#_(defn list-all
```

Comment:
Don't need to leave code commented out. you can remove it, and use git history to refer to it.